docs/home/benefits-of-pwa.md

```diff
@@ -0,0 +1,65 @@
+# Benefits of Progressive Web Apps
+
+Building a Progressive Web App has plenty of advantages, from both a developer and user perspective. If you're looking for a summary of what PWAs can bring to the table, the following article covers several benefits, broken down by how they affect either the development experience or the user experience.
+
+If you are unclear on what a Progressive Web App is from a technical perspective, feel free to check out our [Beginner's Guide to Progressive Web Apps.](/home/pwa-intro/)
+
+## Developer
+
+### Multiple Platforms - Single Codebase
+
+Progressive Web Apps are built with web technology, and as a result, run on any platform with minimal changes to a single, primary codebase. If you develop your app with responsive display design in mind, the same code base will work just as well any mobile (Android, iOS) and any native (Windows, OSX) platform.
```

Comment:
Potential Edit: The same code base will work just as well on mobile, tablet and desktop devices, on any operating system.